### PR TITLE
[FIXED JENKINS-32648] log the name of the node.

### PR DESCRIPTION
**Pull request by jtnord**
When running print the name of the node that executes the remote command.

[JENKINS-32648](https://issues.jenkins-ci.org/browse/JENKINS-32648)
@reviewbybees

### CODE REVIEW

**Comment by ghost** [comment]
This pull request originates from a [CloudBees](https://www.cloudbees.com/) employee. At CloudBees, we require that all pull requests be reviewed by other CloudBees employees before we seek to have the change accepted. If you want to learn more about our process please see [this explanation](https://github.com/reviewbybees/about#about-reviewbybees).

File: src/main/java/hudson/plugins/distfork/DistForkCommand.java [hunk 1]
```diff
@@ -124,6 +124,8 @@ public void run() {
                 try {
                     Computer c = Computer.currentComputer();
                     Node n = c.getNode();
+                    String nodeName = n.getNodeName().equals("") ? "master" : n.getNodeName();
```
Comment (mslusarczyk) [inline src/main/java/hudson/plugins/distfork/DistForkCommand.java:127]: Can n.getNodeName() be null?

Comment (jtnord) [inline src/main/java/hudson/plugins/distfork/DistForkCommand.java:127]: no.  it is `@NonNull` [Javadoc](http://javadoc.jenkins-ci.org/hudson/model/Node.html#getNodeName%28%29)

File: src/test/java/hudson/plugins/distfork/DistForkCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    @Test
+    public void testRunOnMaster() throws Exception {
+        // whoami seems to be in both windows, linux and osx.
+        String result = commandAndOutput("dist-fork", "-l", "master", "whoami");
+        assertThat(result, allOf( containsString("Executing on master"), containsString(System.getProperty("user.name") )));
+    }
+
+    @Test
+    public void testRunOnSlave() throws Exception {
```
Comment (oleg-nenashev) [inline src/test/java/hudson/plugins/distfork/DistForkCommandTest.java:54]:
``` java
@Test
@Issue("JENKINS-32648")
public void shouldPrintExecutingOnSlaveMessageToLogs() throws Exception
```

**Comment by oleg-nenashev** [comment]
Would be also useful to have a test for runs on master.
LGTM by the way

**Comment by jtnord** [comment]
> Would be also useful to have a test for runs on master. 

@oleg-nenashev that is what [`testRunOnMaster()`](https://github.com/jenkinsci/distfork-plugin/pull/5/files#diff-67f2a5d743e79c27edbd388fd452e5efR47) does.

If you know of a way to check that the task actually ran on a specific node I will add it (but I was not aware of one for a Task as opposed to a Run)

**Comment by jtnord** [comment]
@reviewbybees done

**Pull request merged**